include/alpaka/rand/Philox/PhiloxBaseCudaArray.hpp

```diff
@@ -0,0 +1,82 @@
+/* Copyright 2021 Jiri Vyskocil
+ *
+ * This file is part of alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+#include <alpaka/meta/CudaVectorArrayWrapper.hpp>
+
+#if defined(ALPAKA_ACC_GPU_HIP_ENABLED) || defined(ALPAKA_ACC_GPU_CUDA_ENABLED)
+
+namespace alpaka
+{
+    namespace rand
+    {
+        namespace engine
+        {
+            namespace traits
+            {
+                template<typename TScalar>
+                struct PhiloxResultContainerTraits
+                {
+                    using type = void;
+                };
```

Comment:
I would prefer to either not define the type (just declare), or put a `static_assert` inside.